3-Register/src/app/register/page.tsx

```diff
@@ -0,0 +1,27 @@
+import { RegisterForm } from "@/components/RegisterForm";
+import { Ubuntu } from "next/font/google";
+import Link from "next/link";
+
+const ubuntu = Ubuntu({ subsets: ["latin"], weight: ["400", "700"] });
+
+export default function Register() {
```

Comment:
We normally use arrow functions
```suggestion
export const Register = () => {
```

Reply:
A slight tweak: next demands that some components (the ones in "special" files, like `page.tsx`, or `loading.tsx` amongst others) be exported by default. So this could be
`const Register = () => {...}`
`export default Register;`

Reply:
Good to know 👍